libs/shared/typescript/charts/src/lib/x-axis-label-tooltips/x-axis-label-tooltips.ts

```diff
@@ -0,0 +1,66 @@
+import { ECharts } from 'echarts';
+
+/**
+ * Manually add tooltip to chart x-axis labels.
+ * See https://github.com/apache/echarts/issues/19616 and https://github.com/apache/echarts/issues/11903/
+ * Based on https://codepen.io/plainheart/pen/jOGBrmJ.
+ */
+export class XAxisLabelTooltips {
```

Comment:
ECharts doesn't currently support tooltips on x-axis labels, so manually add them to the plot.